contributing.md

```diff
@@ -118,3 +118,36 @@ explanation about the NetworkManager API.
     podman stop lsr-ci-c7
     podman rm lsr-ci-c7
     ```
+
+## Running CI Tests Locally
+
+### Use tox-lsr with qemu
+
+The latest version of tox-lsr supports qemu testing. https://github.com/linux-system-roles/tox-lsr#qemu-testing
+
+**Steps:**
+
+1. If you are using RHEL or CentOS, enable the EPEL repository for your platform - https://docs.fedoraproject.org/en-US/epel/
+
+2. Use yum or dnf to install `standard-test-roles-inventory-qemu`
+   - If for some reason dnf/yum do not work, just download the script from https://pagure.io/standard-test-roles/raw/master/f/inventory/standard-inventory-qcow2 - copy to your `$PATH`, and make sure it is executable
+
+3. Install tox
+   - Use yum/dnf to install `python3-tox` - if that does not work, then use `pip install --user tox`, then make sure `~/.local/bin` is in your `$PATH`
+
+4. Install tox-lsr https://github.com/linux-system-roles/tox-lsr#how-to-get-it
+   ```bash
+   pip install --user git+https://github.com/linux-system-roles/tox-lsr@main
+   ```
+
+5. Assuming you are in a git clone of a role repo which has a tox.ini file - you can use e.g.
+
+   ```bash
+   tox -e qemu-ansible-core-2.14 -- --image-name rhel-9-2 tests/tests_default.yml
```

Comment:
Upstream users wouldn't be able to use rhel, change to centos-9

Reply:
I changed it to centos-9, thanks!